docs/sources/enterprise/auditing.md

```diff
@@ -0,0 +1,166 @@
+
+**Sessions**
+
+- Login
+- Logout
+- Revoke a user authentication token
+- Create or delete an API key
+
+**User management**
+
+- Create, update or delete a user
+- Enable or disable a user
+- Manage user role and permissions
+- LDAP sync or information access
+
+**Team and organization management**
+
+- Create, update or delete a team / organization
+- Add or remove a member to a team / organization
+- Manage team / organization members roles
+- Invite an external member to an organization
+- Revoke a pending invitation to an organization
+- Add or remove an external group to sync with a team
+
+**Folder and dashboard management**
+
+- Create, update or delete a folder
+- Manage folder permissions
+- Create, import, update or delete a dashboard
+- Restore an old dashboard version
+- Manage dashboard permissions
+
+**Data sources management**
+
+- Create, update or delete a data source
+- Manage data source permissions
+
+**Alerts and notification channels management**
+
+- Create, update or delete a notification channel
+- Test an alert or a notification channel
+- Pause an alert
+
+**Reporting**
+
+- Create, update or delete a report
+- Update reporting settings
+- Send reporting email
+
+**Annotations, playlists and snapshots management**
+
+- Create, update or delete an annotation
+- Create, update or delete a playlist
+- Create or delete a snapshot
+
+## Configuration
+
+> The auditing feature is disabled by default.
+
+Audit logs can be saved into files, sent to a Loki instance or sent to the Grafana default logger. By default, only the file exporter is enabled.
+You can choose which exporter to use in the [configuration file](https://grafana.com/docs/grafana/latest/administration/configuration).
+
+Options are `file`, `loki` and `console`. Use spaces to separate multiple modes, e.g. `file loki`.
+
+By default, when a user create or update a dashboard, its content will not appear in the logs as it can significantly increase the size of your logs. If this is important information for you and you can handle the amount of data generated, you can enable this option in the configuration.
+
+```ini
+[auditing]
+# Enable the auditing feature
+enabled = false
+# List of enabled loggers
+loggers = file
+# Keep dashboard content in the logs (request or response fields); this can significantly increase the size of your logs.
+log_dashboard_content = false
+```
+
+Each exporter has its own configuration fields.
+
+### File exporter
+
+Audit logs are saved into files. You can configure the folder to use to save these files. Logs are rotated when the file size is exceeded and at the start of a new day.
+
+```ini
+[auditing.logs.file]
+# Path to logs folder
+path = data/log
+# Maximum log files to keep
+max_files = 5
+# Max size in megabytes per log file
+max_file_size_mb = 256
+```
+
+### Loki exporter
+
+Audit logs are sent to a [Loki](https://grafana.com/oss/loki/) service.
+
+```ini
+[auditing.logs.loki]
+# Set the url for writing logs to Loki
```

Comment:
```suggestion
# Set the URL for writing logs to Loki
```

Reply:
Please change this in the .ini files as well.

Reply:
Any reason that TLS and URL are not capitalized in the fields?

Reply:
It follows the convention used for the Grafana configuration file.

Reply:
Most of which was written before I got here. We're updating it where we can, but the current convention in documentation is to update URL in all cases except for where it would mess up the code. If you can change it for the comments, that would be great. If not, then it is not a huge deal.

Reply:
Sure, I updated the comment and will update the .ini files in an Enterprise PR. My answer was for your question on the fields: the convention is to not use capitalized letters in the configuration fields.